plugins/local_file/src/interface.h

```diff
@@ -0,0 +1,10 @@
+#ifndef USER_MAPPER_INTERFACE
+#define USER_MAPPER_INTERFACE
+
+extern "C" {
+  void* user_manager_init(const char* _args);
+  char* user_manager_match(void* _manager, const char* _param);
+  void user_manager_close(void* _manager);
```

Comment:
Is `user_manager_` a placeholder for now?

Reply:
Different names are welcome. Otherwise, it'll be left as is.

Reply:
Replace manager in name with:
- ~map~
- mapper
- ~mapping~

Reply:
Done.